README.md

```diff
@@ -45,6 +45,12 @@ A few things to notice:
 
 ![](doc/images/tiled_screenshot_1.png)
 
+If you have exit scene
+- You must create layer "exit". The layer have cases where the gamer can switch to the next scene.
+- In layer properties, you must add "exitSceneUrl" property. It represents a url map of the next scene. Be careful, if you want that the next map will be correctly loaded, you must check that the map files are in folder  `back/src/Assets/Maps/<your map folder>`. The files will be accessible by url `<HOST>/map/files/<your map folder>/...`.
```

Comment:
Wait... if a layer must be called "exit" and must have a "exitSceneUrl", that means one scene can only have one exit, right? Shouldn't we change this?

Reply:
No, you can create two layers with the same name "exit" (the id layer will be different) and add multi tile to exit. 
One tile cannot behave the param "exitSceneUrl" because in this case, yes, one tile = one exit and in the TileSet we should have each tileset to each exit scene 👎.
To have multi exit on the map, we must use the layers with multi tileset what represents exit case for the gamer.
So one layer = one scene exit, but we could have infinite "exit" layers on the map.

Reply:
:+1: